### PR TITLE
Added Swagger and Cors middleware

### PR DESCRIPTION
**Pull request by jackdevitt**
### Features
- Added Swagger documentation available at <host>/swagger/index.html
- Added CORS middleware to make API usable with web browser

### Media
Screenshot 2023-07-24 at 11 01 08 AM

### Moving forward
- Hook up to searchbar

### CODE REVIEW

File: readme.md
Comment (matt-j-deasy) [inline readme.md:1]: Idk if this is a Go thing, but in most projects README is capitalized

File: movie.go [hunk 1]
```diff
@@ -78,6 +86,18 @@ func getMovieWithDetail(id int) MovieDetails {
 				} else {
 					var actors Actors
 					json.Unmarshal(body, &actors)
+
+					var crew Crew
+					json.Unmarshal(body, &crew)
+
+					for i := 0; i < len(crew.EntireCrew); i++ {
+						fmt.Println(crew.EntireCrew[i].Name + " " + crew.EntireCrew[i].Job)
+						if crew.EntireCrew[i].Job == "Producer" || crew.EntireCrew[i].Job == "Executive Producer" {
```
Comment (matt-j-deasy) [inline movie.go:95]: Am I correct in assuming this is a massive list?

File: readme.md
Comment (jackdevitt) [reply readme.md:1]: good to know, I will fix it now

File: movie.go (hunk 1 above)
Comment (jackdevitt) [reply movie.go:95]: It depends on the movie, for some movies the list is barely 5 people, but for some of the bigger movies the list can go so far as about 100-200 people (don't quote me on exact numbers)

Comment (jackdevitt) [reply movie.go:95]: Nice thing is producers usually appear at the top so even massive lists tend to only have a few iterations

**Review by matt-j-deasy — APPROVED** [review]

**Pull request merged**